CSS/CSS.sublime-syntax

```diff
@@ -603,14 +683,16 @@ contexts:
               | hyphens|overflow-scrolling|overflow|color-profile|kerning
               | nbsp-mode|color|image-resolution|grid-row|grid-column|blend-mode
               | azimuth|pause-after|pause-before|pause|pitch-range|pitch|text-height
+              | system|negative|prefix|suffix|range|pad|fallback|additive-symbols
+              | symbols|speak-as
             )\b
           scope: support.type.property-name.css
     - match: (:)\s*
       captures:
         1: punctuation.separator.key-value.css
       push:
-        - meta_scope: meta.property-value.css
-        - match: '\s*(;|(?=\}))'
+        - meta_content_scope: meta.property-value.css
+        - match: '\s*(;)|(?=[})])'
```

Comment:
I added `)` to the lookahead here to keep things simple for `@supports`. This now means that rules can be improperly terminated by `)` like so:

```css
.c {
    background: none)background: none
}
```

Should I duplicate the necessary [`rule-list-body`](https://github.com/Briles/Packages/blob/fdf35f282921b3532db9b2359820655c70307c99/CSS/CSS.sublime-syntax#L598) constructs so this improper termination doesn't happen?